examples/simple/Error_Codes.csv

```diff
@@ -48,109 +48,108 @@ UB-CEER1,Trying to read through a null pointer.,"6.3.2.1:1, J.2:1 item 19",Undef
+UB-TDL5,Declaration of an identifier for an object with no linkage and incomplete type.,"6.7:7, J.2:1 item 59",Undefined Behavior
+UB-CDT1,Structs must contain a named member.,"6.7.2.1:8, J.2:1 item 61",Undefined Behavior
+UB-CDT2,Unions must contain a named member.,"6.7.2.1, J.2:1 item 61",Undefined Behavior
+UB-TDG1,Declaration for the function appears with type qualifiers.,"6.7.3:9, J.2:1 item 66",Undefined Behavior
+UB-TDG3,Declaration for Id appears with invalid storage class specifiers.,"6.7.1:7, J.2:1 item 60",Undefined Behavior
+UB-TEAS4,Void value as right operand in assignment,"6.3.2.2:1, J.2:1 item 23",Undefined Behavior
+UB-TDR1,Declaration for inline function not defined in the same translation unit.,"6.7.4:7, J.2:1 item 70",Undefined Behavior
+UB-TDR2,No definition for symbol with external linkage.,"6.9:5, J.2:1 item 84",Undefined Behavior
 UB-TIN1,Identifier declared with incompatible types in different translation units.,"6.2.7:2, J.2:1 item 15",Undefined Behavior
 UB-TIN2,Multiple external definitions for Identifier.,"6.9:5, J.2:1 item 84",Undefined Behavior
-UB-FD4,definition of main requires a prototype.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
-UB-FD5,main must return an int.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
-UB-FD6,"if main has arguments, the type of the first argument must be equivalent to int.","5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
-UB-FD7,"if main has arguments, the type of the second argument must be equivalent to char**.","5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
-UB-FD8,main must have zero or two parameters.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
-UB-FD9,main must have zero or two parameters.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
+UB-FD4,Definition of main requires a prototype.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
+UB-FD5,Function main must return an int.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
+UB-FD6,"If main has arguments, the type of the first argument must be equivalent to int.","5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
+UB-FD7,"If main has arguments, the type of the second argument must be equivalent to char**.","5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
+UB-FD8,Function main must have zero or two parameters.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
+UB-FD9,Function main must have zero or two parameters.,"5.1.2.2.1:1, J.2:1 item 4",Undefined Behavior
 UB-CTI1,Arrays must have integer length.,"6.6:6, J.2:1 item 55",Undefined Behavior
-UB-STDARG1,"'va_start' used in function with fixed args.","6.9.1:8, J.2:1 item 87",Undefined Behavior
-UB-STDIO1,"printf: Format string expects type of size a, but got an argument of size b.","7.21.6.1:9, J.2:1 item 153",Undefined Behavior
+UB-STDARG1,'va_start' used in function with fixed args.,"6.9.1:8, J.2:1 item 87",Undefined Behavior
+UB-STDIO1,"'printf': Format string expects type of size a, but got an argument of size b.","7.21.6.1:9, J.2:1 item 153",Undefined Behavior
```

Comment:
Are size "a" and size "b" correct here?  Are they replaced with actual sizes when the error message is reported?